public/js/forgotPassword.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * This file defines logic for the forgot password page.
+ */
+
+var $submitForm = $('#submitForm');
+var $emailField = $('#submitForm input[type="email"]');
+var $message = $('.message');
+
+// Display a confirmation message when the email is submitted.
+$submitForm.submit(function (evt){
+	evt.preventDefault();
+
+	// Display a waiting message.
+	$message.html(STATUS_MESSAGE.WAITING);
+
+	// Make the request for a reset email.
+	$.post('/forgot', $(this).serialize(), function (data){
+		if (data.success){
+			// Display the success message.
+			$message.html(STATUS_MESSAGE.SUCCESS);
+			$submitForm.slideUp();
+		} else {
+			// Display the failure message.
+			$message.html(STATUS_MESSAGE.FAILURE);
+		}
+
+	}).fail(function (data){
+		$message.html(STATUS_MESSAGE.FAILURE);
+		
+	}).always(function (){
+		$emailField.focus();
+	});
+});
+
+
+// Define status messages to be displayed.
+var STATUS_MESSAGE = {
+	WAITING: 'Please wait, as we send you a reset email...',
```

Comment:
Similar nit to above- I think we can improve the comma here as well and have it read as one phrase. What do you think @psybuzz ?